src/main/java/taxi/controller/page/CreateDriverController.java

```diff
@@ -0,0 +1,53 @@
+package taxi.controller.page;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {
+    private static final String LOGIN_PAGE = "/login";
+    private static final String SALT = "salt";
```

Comment:
It was not part of the task, we will learn how to properly store sensitive data later on, so let's remove it for now